api/authCustom/views.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework.decorators import api_view, parser_classes
+from rest_framework.parsers import JSONParser
+from rest_framework.response import Response
+from django.utils import timezone
+
+from .models import Token_Custom
+from .utils import env
+
+
+
+@api_view(["POST"])
+@parser_classes([JSONParser])
+def goal_auth_token(request, userId):
+    try:
+        rest_key = request.data.get('rest_key')
+        if rest_key != env('REST_KEY'):
+            raise Exception("Wrong REST_KEY")
+
+        try:
+            token = Token_Custom.objects.get(userId=userId)
+            if token.is_invalid(createdTime=token.created.timestamp()):
```

Comment:
Passing `createdTime` doesn't feel right here. Shouldn't it be already available in token self scope?